vimeo/VIMVideoPlayer
# Adds framework project as source container

#### Ticket
No ticket

#### Ticket Summary
Source files were being compiled straight in the example project. Is there an alternative?

#### Implementation Summary
This pull request adds a framework project to the workspace so that the library can be compiled and worked on in the context of the framework project. And then consumed in the example project. 

#### How to Test
Open the workspace. 

Build the framework project. Confirm there are no warnings or errors. 

Build and run the example project. Confirm there are no warnings or errors. Confirm it functions properly in the simulator and on device.
